include/ocpp/v201/device_model.hpp

```diff
@@ -210,10 +200,7 @@ class DeviceModel {
     /// \param component_variables
     /// \param component_criteria
     /// \return
-    std::vector<ReportData>
-    get_report_data(const std::optional<ReportBaseEnum>& report_base = std::nullopt,
-                    const std::optional<std::vector<ComponentVariable>>& component_variables = std::nullopt,
-                    const std::optional<std::vector<ComponentCriterionEnum>>& component_criteria = std::nullopt);
+    std::vector<ReportData> get_base_report_data(const std::optional<ReportBaseEnum>& report_base = std::nullopt);
```

Comment:
Does this still need to be an optional? I assume it is now an always required variable here.